config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,23 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func isMountpointForbidden(path string) bool {
+	// Check if the path is a restricted directory
+	restrictedDirs := []string{"/boot", "/sys", "/usr", "/var", "/", "/sysroot/mnt"}
+	for _, dir := range restrictedDirs {
+		if path == dir {
+			return true
+		}
+	}
+	return false
+}
+
+func (c Config) ValidateMountPoints() (r report.Report) {
```

Comment:
WDYT about calling this `validate()`?

Reply:
i was thinking about that but wouldn't that mess with [](https://github.com/coreos/butane/pull/436)?

Reply:
because then there would be two validate methods that use config

Reply:
That is a good question, you will have a conflict once one or the other merges, but that should not impact the naming, the conflict should be fairly easy to solve, and the validate() can include both sections you are validating